test/regression/cases/file_tree/datadog-agent/datadog.yaml

```diff
@@ -7,7 +7,10 @@ cloud_provider_metadata: []
 
 logs_enabled: true
 
-dd_url: http://127.0.0.1:9092
-telemetry.enabled: true
-telemetry.checks: '*'
-process_collection.enabled: false
+dd_url: http://127.0.0.1:9091
+telemetry:
+  enabled: true
+  checks: '*'
+process_config:
```

Comment:
Is there a particular reason why you're favoring the nested structure vs `key.otherkey: value`?